### ISSUE TITLE
adding TomoScan2BM class

### ISSUE THREAD

**Issue by decarlof** [tomography/tomoscan#21]
I will be opening a pull request, please keep it open till I am done. Before committing we need to decide if some components should go into the TomoScan class (for example the handling of the FrameType etc.)

**Comment by MarkRivers** [comment]
OK, sounds good.

FrameType probably should go in the base class, I was meaning to ask you about that.

**Comment by MarkRivers** [comment]
I have got the current version running on the Windows machine at 13-BM-D and collecting actual data!  I'm pretty happy with this design.  I hope it's working for you too.

**Comment by MarkRivers** [comment]
I have changed the base class to put all of the scan parameters into class variables that get set from the current PV values in begin_scan().  This has 2 advantages:
- The derived class does not need to access the PVs, just the class variables. This is simpler and cleaner.
- It prevents things from messing up if the user accidentally changes a scan parameter during a scan.  The values are captured at the beginning of the scan, and so it does not matter if the user changes them during the scan.

I have modified tomoscan_13bm.py to save all data in a single file.  I have added commented out code that shows how to save in multiple files if desired.  I have deleted tomoscan_13bm_single_file.py because it is now redundant.

I have tested the new code in tomoscan_13bm.py. 

I will make a new fix_2bm branch and fix tomoscan_2bm.py to use the class variables and fix the other problems I mentioned in my review of the pull request.  I will then make a pull request for the fix_2bm branch.
